frontend/src/components/DefenceBox/DefenceMechanism.css

```diff
@@ -1,106 +1,164 @@
-.defence-mechanism {
-	cursor: default;
+.defence-mechanism-fieldset {
+	min-height: fit-content;
+	margin: 0.1em;
```

Comment:
I feel like this margin is so small that it may as well not be there?

Reply:
In fact, I noticed a tiny tiny gap between the edge of the border and the edge of the defence-configuration box, which I'd prefer to not be there. Or if we want a gap, then I reckon it should be a bit bigger so that it looks more intentional.

![image](https://github.com/ScottLogic/prompt-injection/assets/118171430/8576aac2-0a02-4d0f-a428-03f464f1c04e)

So based on that, I'd actually prefer to have `margin: 0` to overwrite these tiny margins being added to the field set by 'user agent stylesheet' (what is that?).

![image](https://github.com/ScottLogic/prompt-injection/assets/118171430/e2629e31-64b7-4747-aaca-06d43156eb57)

![image](https://github.com/ScottLogic/prompt-injection/assets/118171430/f197f688-b0ab-4dd0-8e51-8761a2f237d0)

Reply:
yep, agreed, I'll turn this to margin: 0;